curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-a-template-engine.english.md

```diff
@@ -15,9 +15,9 @@ A template engine enables you to use static template files (such as those writte
 
 To set up <em>Pug</em> for use in your project, you will need to add it as a dependency in your package.json. Don't forget to add the name of the package and the version. Use the package.json for some examples of the correct syntax.
 
-Express needs to know which template engine you are using. We will use the <b>set</b> method to assign 'pug' as the <b>'view engine'</b>. <code>app.set('view engine', 'pug')</code>
+Express needs to know which template engine you are using. We will use the <code>set</code> method to assign <code>pug</code> as the <code>view engine</code>: <code>app.set('view engine', 'pug')</code>
 
-Your page will not load until you correctly <em>render</em> the index file in our <em>'views/pug'</em> directory.
+Your page will not load until you correctly <code>render</code> the index file in our <code>views/pug</code> directory.
```

Comment:
Whilst we are doing this, could you also change `our` to `the` please Nicholas? If that does not sound nice to you, then another option could be `your`

Reply:
Taken care of 🙂. Thanks for the catch!